raiden/udp_message_handler.py

```diff
@@ -117,21 +122,30 @@ def handle_message_lockedtransfer(raiden: 'RaidenService', message: LockedTransf
         raiden.mediate_mediated_transfer(message)
 
 
+def handle_message_processed(raiden: 'RaidenService', message: Processed):
```

Comment:
Add a docstring here saying that returning `True` would be a message to the sender informing them it's fine to send a `Delivered` mesage back to the partner.

Reply:
done